src/routes/(docs)/docs/content/v4/maintenances/rrule-patterns.md

```
@@ -619,4 +645,4 @@ Description: |
 
 - [Creating and Managing Maintenances](/docs/v4/maintenances/creating-managing) - Apply these patterns in the dashboard
 - [Maintenance Events](/docs/v4/maintenances/events) - Understand how RRULE generates events
-- [Maintenance Overview](/docs/v4/maintenances/overview) - Learn maintenance fundamentals
+- [Maintenance Overview](/docs/v4/maintenances/overview) - Learn maintenance fundamentals
```

markdownlint-cli2 (0.23.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)